internals-js/src/error.ts

```diff
@@ -301,6 +301,11 @@ const INVALID_LINK_DIRECTIVE_USAGE = makeCodeDefinition(
   'An application of the @link directive is invalid/does not respect the specification.'
 );
 
+const REQUIRED_ARGUMENT_MISSING_IN_SOME_SUBGRAPH = makeCodeDefinition(
+  'REQUIRED_ARGUMENT_MISSING_IN_SOME_SUBGRAPH',
+  'An argument (of a field or directive definition) is not defined in all the subgraphs defining the field and is mandatory in at least some of them.'
```

Comment:
**nitpick** I think this error message could be more clear (but that might be due to my newness)

```suggestion
  'An argument of a field or directive definition is mandatory in some subgraphs, but the argument is not defined in all subgraphs that define the field or directive definition.'
```